mappings/17b644266c4187aab510d8f6825aa12c66bfc4a12d0aa2bc5a4581ad50414c4d2045636f6e6f6d7920546f6b656e.json

```diff
@@ -0,0 +1,64 @@
+{
+    "subject": "17b644266c4187aab510d8f6825aa12c66bfc4a12d0aa2bc5a4581ad50414c4d2045636f6e6f6d7920546f6b656e",
+    "url": {
+        "sequenceNumber": 0,
+        "value": "https://palmeconomy.io/",
```

Comment:
Hi @vorujack,

can you please point to the place on this page where policyId of the submitted token is located? If there is none, please put it so we can verify the project.

Thanks!

Reply:
Hi.
Added in tokenomic section